content/posts/May-2022.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "May 2022"
+date: 2022-05-02T10:01:16+01:00
+weight: 62
+---
+
+## Week 17 (April 26th - May 2th)
+
+- We have fixed a bug in ogr that caused `IssueTrackerDisabled` being raised only when trying to create an issue on git project with disabled issue tracker. Now it is also raised when getting a specific issue or an issue list.
+  ([ogr#703](https://github.com/packit/ogr/pull/703))
+- `ogr` now correctly raises `OgrException` when given invalid URL to `PagureService.get_project_from_url`.
+  ([ogr#705](https://github.com/packit/ogr/pull/705))
+- When using Tokman as GitHub authentication mechanism, ogr will now raise `GithubAppNotInstalledError` instead of failing with generic `GithubAPIException` when app providing tokens is not installed on the repository.
+  ([ogr#706](https://github.com/packit/ogr/pull/706))
+- The packaging issue with missing `pkg_resources` is fixed. It now uses the standard library for getting the version on Python 3.8+, or a smaller package on older Pythons.
+  ([ogr#710](https://github.com/packit/ogr/pull/710))
```

Comment:
I think we can skip this one since it's not relevant to Packit users. (I've added that entry just for the OGR changelog.)

```suggestion
```